modules/mobile/app/controllers/mobile/v0/decision_letters/downloads_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'claim_letters/claim_letter_downloader'
+
+module Mobile
+  module V0
+    module DecisionLetters
+      class DownloadsController < ApplicationController
+        before_action { authorize :bgs, :access? }
+
+        def show
+          document_id = CGI.unescape(params[:document_id])
+
+          service.get_letter(document_id) do |data, mime_type, disposition, filename|
+            send_data(data, type: mime_type, disposition: disposition, filename: filename)
```

Comment:
in the other letters download endpoint, we're hardcoding the disposition to `attachment`. I confirmed with the benefits team that we should only be getting a mime type of pdf but unsure if disposition will change or exactly what disposition does. Wondering if we should hardcode it as well. in the specs does it come in as `attachment`?

Reply:
Ya it is `attachment`. Is it worth hardcoding then?

Reply:
Cool, probably safe to just leave it as it is.